### PR TITLE
Add Tutorial: Mount Hetzner Cloud volume on multiple Debian servers with `sshfs`, including optional Ansible role

### PR DESCRIPTION
**Pull request by tlrh314**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Timo Halbesma <timo@halbesma.com>

### CODE REVIEW

File: tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+```
+
+### Step 3.2 - Setup the (root) ssh key and config on `<server2>`, and accept it on `<server1>`
+
+We are now going to configure `sshfs` to mount the volume from `<server1>` to `<server2>`. 
+
+```bash
+# Reset and echo the root passwords
+echo "The root password of server1 is: $(hcloud server reset-password server1 | sed 's/.*\(reset to: \)//')"
+echo "The root password of server2 is: $(hcloud server reset-password server2 | sed 's/.*\(reset to: \)//')"
```
Comment (n-se) [inline tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md:107]: Why do you reset the password here? You used SSH keys on creation and below you also only use SSH keys.

File: tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+ansible-playbook provision.yml
+
+# Destroy the setup at Hetzner Cloud
+./create.sh -d
+```
+
+## Limitations
+- This tutorial limits the scope to 2 servers only, but you can easily extend this to N servers.
+- The order of (re)booting the servers is important! If `<server1>` is not reachable from `<server2>` on boot, then the volume will of course not be mounted! The described solution is thus not entirely fool-proof and caution is required when (re)booting your servers. In some cases a manual mount may be needed.
+- This setup requires `PermitRootLogin yes` in `/etc/ssh/sshd_config` of server1, which may not be ideal.
```
Comment (n-se) [inline tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md:214]: without-password instead of yes should be better and is more secure

**Review by n-se — COMMENTED** [review]
Great tutorial! I have added a few notes

File: tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md (hunk 1 above)
Comment (tlrh314) [reply tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md]: I have removed the `reset-password` call in 6086cd349cbcc5b4e3a83eb0004bcd7746f95b09.

File: tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md (hunk 2 above)
Comment (tlrh314) [reply tutorials/mount-volume-on-multiple-servers-with-sshfs/01.en.md:210]: I added a caveat in 4bf90becd72c3b50c63f4f39759cfd753866562e stating that this tutorial configures the server with minimum requirements for the `sshfs` mount, and refer to [Securing SSH tutorial](https://community.hetzner.com/tutorials/securing-ssh) for securing the SSH service. 

The suggested change is out of scope for the `sshfs` tutorial in my view, but if you think the `sshfs` tutorial would indeed benefit from setting `PasswordAuthentication no` in `sshd_config` then I can update this is the tutorial as well as the Ansible playbook.

**Comment by tlrh314** [comment]
> Great tutorial! I have added a few notes

Awesome, many thanks for your review! I pushed some commits to implement the suggested changes and marked all notes as resolved. 

If you think further updates for are https://github.com/hetzneronline/community-content/pull/250#discussion_r476392615 are required I'm happy to update the tutorial and Ansible playbook.

**Review by n-se — APPROVED** [review]
LGTM

**Pull request merged**